flink-kubernetes/src/main/java/org/apache/flink/kubernetes/configuration/KubernetesConfigOptions.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.kubernetes.configuration;
+
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.kubernetes.cli.KubernetesCliOptions;
+
+import static org.apache.flink.configuration.ConfigOptions.key;
+
+/**
+ * This class holds configuration constants used by Flink's kubernetes runners.
+ */
+public class KubernetesConfigOptions {
+
+	public static final ConfigOption<String> REST_SERVICE_EXPOSED_TYPE =
+		key("kubernetes.rest-service.exposed.type")
+		.stringType()
+		.defaultValue(ServiceExposedType.LoadBalancer.toString())
+		.withDescription("It could be ClusterIP/NodePort/LoadBalancer(default). When set to ClusterIP, the rest service" +
+				"will not be created.");
+
+	public static final ConfigOption<String> JOB_MANAGER_SERVICE_ACCOUNT =
+		key("kubernetes.jobmanager.service-account")
+		.stringType()
+		.defaultValue("default")
+		.withDescription("Service account that is used by jobmanager within kubernetes cluster. " +
+			"The job manager uses this service account when requesting taskmanager pods from the API server.");
+
+	public static final ConfigOption<Double> JOB_MANAGER_CPU =
```

Comment:
I do not suggest to unify the cpu configuration. For Yarn cluster, we use the vcores instead of cpu to allocate from Yarn ResourceManager. Usually one vcore represent one cpu. However, it it not always true. And it only support integer number. For kubernetes cluster, we directly allocate cpu from ApiServer. The cpu is a `double` value, that means we could allocate 0.1 cpu. 

So i suggest to separate the config options now. Maybe in the future, we could find some ways to unify the cpu configurations.

Reply:
Ok. I think it's fine to have separate configs for now since CPU is not a common config for all resource framework yet.